packages/teleport-shared/src/node-handlers/node-to-html/types.ts

```diff
@@ -15,6 +15,7 @@ export interface HTMLTemplateSyntax {
   interpolation?: (value: string) => string
   valueBinding?: (value: string, node?: UIDLElementNode) => string
   eventBinding?: (value: string) => string
+  eventNameBinding?: (value: string) => string
```

Comment:
I'd find a different name for this, this is just used to add the function call parentheses right? how about: `eventHandlersBindingMode`: 'reference' | 'call' and maybe leave some explanations

Reply:
Regarding this, we can't use reference, callee pattern. Here is the reason, if we combine and make it as a single `templateSyntax.eventBindingMode` which return 
```
@click="handleClick"
```
or
```
(click)="handleClick()"
```
according to syntax which looks clean but the *prettyHTML* which we use will replace special characters and spaces.
so it ends up generating 
```
(click)<#&..some constants>"handleClick"
```